src/components/AnchorForCommentsOnly/BaseAnchorForCommentsOnly/index.native.js

```diff
@@ -3,6 +3,12 @@ import {Linking, StyleSheet} from 'react-native';
 import {propTypes, defaultProps} from '../anchorForCommentsOnlyPropTypes';
 import fileDownload from '../../../libs/fileDownload';
 import Text from '../../Text';
+import PressableWithSecondaryInteraction from '../../PressableWithSecondaryInteraction';
+import {showContextMenu} from '../../../pages/home/report/ContextMenu/ReportActionContextMenu';
+import {contextMenuTypes} from '../../../pages/home/report/ContextMenu/ContextMenuActions';
+
+const linkRef = React.createRef();
```

Comment:
Our style guide dictates that we should prefer the use of [callback refs](https://reactjs.org/docs/refs-and-the-dom.html#callback-refs)

Reply:
Please move this inside the component.